### ISSUE TITLE
konga healthcheck

### ISSUE THREAD

**Issue by lukasa1993** [pantsel/konga#379]
could you please add health-check to konga?

**Comment by pantsel** [comment]
What is it exactly that you want to do?

If you have installed Konga from docker, you can add healthchecks on your docker config file to check if the container is running.

**Comment by lukasa1993** [comment]
yes i am using docker-compose do you have any example for healthcheck call ? which wont self ddos ?

**Comment by evandam** [comment]
Hey @pantsel, just wanted to bump this. What I would expect is an endpoint like `/status` => `OK` or `/ping` => `PONG`.

The reason for this is when deploying Konga with something like K8s, Nomad, etc., a health check makes sure the service is actually up and reachable before taking traffic.

So far running Konga, there's a bit of time between when the container starts and when it's actually ready to start serving requests, like when it is seeding data.
